omisego-sdk/src/main/java/co/omisego/omisego/model/User.kt

```diff
@@ -19,6 +17,6 @@ import com.google.gson.annotations.SerializedName
  *
  */
 data class User(val id: String,
-                @SerializedName("provider_user_id") val providerUserId: String,
+                val providerUserId: String,
                 val username: String,
-                @SerializedName("meta_data") val metaData: HashMap<String, Any>?)
+                val metaData: HashMap<String, Any> = hashMapOf())
```

Comment:
I think using the constructor makes more sense if you're not setting any mapping
Do you need the field to be `HashMap`? I think `MutableMap` would be enough ? (Or `Map` if it's read-only)

Reply:
I change it to

`val metaData: Map<String, Any>`

without the initialization, because it will be initialized by the converter anyway.